Failure to build VT+checkpoint+detector in TPL directory in app

**Describe the bug**
```
20:17:27  -- Configuring done
20:17:28  CMake Error: install(EXPORT "EMPIRE-targets" ...) includes target "input_deck" which requires target "checkpoint" that is not in this export set, but in multiple other export sets: cmake/checkpointTargets.cmake, cmake/vtTargets.cmake.
20:17:28  An exported target cannot depend upon another target which is exported multiple times. Consider consolidating the exports of the "checkpoint" target to a single export.
20:17:28  CMake Error: install(EXPORT "EMPIRE-targets" ...) includes target "RandomNumberGenerator" which requires target "checkpoint" that is not in this export set, but in multiple other export sets: cmake/checkpointTargets.cmake, cmake/vtTargets.cmake.
20:17:28  An exported target cannot depend upon another target which is exported multiple times. Consider consolidating the exports of the "checkpoint" target to a single export.
20:17:28  CMake Error: install(EXPORT "EMPIRE-targets" ...) includes target "utils" which requires target "checkpoint" that is not in this export set, but in multiple other export sets: cmake/checkpointTargets.cmake, cmake/vtTargets.cmake.
20:17:28  An exported target cannot depend upon another target which is exported multiple times. Consider consolidating the exports of the "checkpoint" target to a single export.
20:17:28  -- Generating done
20:17:28  CMake Generate step failed.  Build files cannot be regenerated correctly.
```

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -339,7 +339,7 @@ export(
                             ${MIMALLOC_LIBRARY_EXPORT}
                             ${FORT_LIBRARY_EXPORT}
                             ${FMT_LIBRARY_EXPORT}
-                            ${CHEKPOINT_EXPORT}
+                            ${CHECKPOINT_EXPORT}
                             ${DETECTOR_EXPORT}
                             ${JSON_LIBRARY}
                             ${BROTLI_LIBRARY}

